.projenrc.js

```diff
@@ -1,4 +1,7 @@
 const { typescript } = require("projen");
+
+const MIN_CDK_VERSION = "2.20.0";
```

Comment:
Do we need to specify the version or can we just use default. Projen defaults to latest.

Reply:
As a library, we should pin to the version we suggest in the peer deps. I was having trouble keeping them all the same version as I added new CDK deps, the latest would be installed, but the older ones were not being upgrading, existentially the one in alpha. Having this constant made it easier to keep them the same version.